docs/install-manual.md

```diff
@@ -66,13 +66,17 @@ This example uses the Docker Hub based `token` discovery service. Log into **eac
 
 		$ docker run -d swarm join --addr=172.31.40.100:2375 token://6856663cdefdec325839a4b7e1de38e8
 
-3. Start the Swarm manager on any machine or your laptop. 
+We are done with the per node setup, and will now set up the manager to control the nodes in the swarm.
```

Comment:
@MHBauer Thanks Morgon...we should remove the royal "We" wherever it occurs.  "You" are the actor in technical documentation.

Um, a new heading here too. Users can then glance at the mini toc to get an overview of the steps.

Sooo ->


##  Configure a manager

Once you have your nodes established, set up a manager to control the swarm.